README.md

```diff
@@ -43,6 +43,10 @@ The following should be present on the target system
 | matrix_synapse_log_days_keep | 30 |
 | matrix_synapse_skip_tls | false |
 | matrix_synapse_registration_secret | _randomly generated_ |
+| matrix_synapse_deployment_method | pip | Either pip or docker [^1] |
+| matrix_synapse_supervision_method | systemd | Either systemd, runit or docker [^1] |
+
+[^1]: Docker must be used for both or neither deployment and supervision
```

Comment:
Please use a proper reference rendered by Github, as this is the primary mean people get their information

Reply:
I thought that would be rendered by github, will fix it